### DISCUSSION TITLE
Will this mod be "True" VR or faked VR using KBM like softwares such as vorpX?

### DISCUSSION THREAD

**Discussion by ghost** [DaXcess/LCVR#7] [Q&A] [answered]
will this be (assuming its possible and will be completed) be like a true VR game with tracked hand movements, grabbing, walking using controllers etc. or will it be a "faked" VR mod that softwares such as vorpX do? e.g you can use the headset to look around in game but still need to move/interact with keyboard and mouse?

**Comment by DaXcess** [comment, marked as answer]
This mod will be (and is already) true VR with hand tracking.
Some controls will still need to be thought out, as currently it is basically a rebinding of the gamepad to the VR controllers, which makes certain VR interactions not possible (e.g. whacking with a stop sign, throwing items, and the new spray paint can).

**Reply by micsthepick** [reply]
what would need to be done to allow KBM control with e.g. PhoneVR, if the user is playing seated and has no vr controllers?

**Reply by DaXcess** [reply]
Rewriting the entire mod is what would need to be done, as everything relies on the hands being tracked (interactions, menus, etc)
